pages/db/migrate/20150720155305_update_slug_and_title_in_refinery_page_parts.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateSlugAndTitleInRefineryPageParts < ActiveRecord::Migration
+  def change
+    Refinery::PagePart.all.each do |pp|
+      unless pp.title
+        pp.title = pp.slug
+      end
+      pp.slug = pp.slug.downcase.gsub(" ", "_")
+      pp.save!
+    end
```

Comment:
it has to be mandatory for the user, i don't know to do it in other way. Do you have any solution?